### ISSUE TITLE
Equations simply vanishing when inserted on tabed text

### ISSUE THREAD

**Issue by Luke2Sky** [RyotaUshio/obsidian-math-booster#173]
When I insert a equation in the following way it and the text close to it are deleted

> \tab Random text `$$\frac{(14)+(15)+3(16)+2(22)+2(24+5(25))}{14}=\frac{294}{14}=21$$`
Note that it has to have a tab before of the text for the bug to happen. 

If I insert only the equation, without text but with a tab the extension works normally:
>\tab `$$\frac{(14)+(15)+3(16)+2(22)+2(24+5(25))}{14}=\frac{294}{14}=21$$`

**Comment by RyotaUshio** [comment]
Thanks, I could reproduce this bug. I will look into it.

**Comment by Luke2Sky** [comment]
I actually don't know if this is a separate bug but I tend to use a very weird notation to have mathblocks closer to one another without matrices which is using them like this: `$$math$$$$math$$`.
This works fine in vanilla obsidian, but it seems the plugin can't handle this madness. I don't blame it lol.

**Comment by RyotaUshio** [comment]
One thing, the text is not actually **deleted** but unintentionally **made invisible**, probably the CSS code of this plugin.
This is why you were able to see it after disabling Math Booster. So I will correct the CSS code.

> `$$math$$$$math$$`

Note that this kind of notation will be problematic for equation numbering, one of the core feature of this plugin.
See the [docs](https://ryotaushio.github.io/obsidian-math-booster/equation-number.html#remarks) for more details.

But as for the original issue, I'm pretty sure it is something that must be resolved.

**Comment by Luke2Sky** [comment]
Yes, I just realized it when I changed to reading mode. I will stop this madness and do things in a better way. One unintentional consequence of this hybrid editing mode is that my source looks more terrible than ever.

**Comment by RyotaUshio** [comment]
Oh, I noticed that `$$math$$$$math$$` also becomes invisible...

**Comment by Luke2Sky** [comment]
I am not sure if I should do a separate issue for this, but I am also finding the following problem:
This is the text in the source mode:
![image](https://github.com/RyotaUshio/obsidian-math-booster/assets/61480539/14a59fc9-eb52-44fb-8274-1c62f1162de9)
In the editing mode it is being hidden:
![image](https://github.com/RyotaUshio/obsidian-math-booster/assets/61480539/570bb0d5-d14c-439c-a323-9dea59a034dd)

**Comment by Luke2Sky** [comment]
Not only that, but if you insert `$$$$` in the middle of a paragraph, the entire paragraph disappears. I think this is actually the same origin to the problem.

**Comment by RyotaUshio** [comment]
Thanks, I tested in the sandbox vault and could reproduce it as well.

https://github.com/RyotaUshio/obsidian-math-booster/assets/72342591/42883e4e-5d5e-463f-8466-af870d87cc84

Fortunately, I've confirmed that this bug is already fixed in v0.7.0 (it's unreleased yet, though).

https://github.com/RyotaUshio/obsidian-math-booster/assets/72342591/a193d2c0-fb49-4514-b257-dd4700826488

**Comment by RyotaUshio** [comment]
But the original `\tab` issue is not resolved.

**Comment by DDDOH** [comment]
Observed the same issue. The below markdown file can reproduce this issue.

```
- Consider both rectilinear distance and Euclidean distance
- $$Z(x)=\max_{w\in W}\max_{y\in \mathbb{R}^2}\frac{F(w,x)}{F(w,y)}$$
- Denote $F^*(w)=\min_{y} F(w,y)$, and $REGR(x,y)=\max_{w\in W}\frac{F(w,x)}{F(w,y)}$
- $Z(x)=\max_{w\in W}\frac{F(w,x)}{F^*(w)}$. Optimal solution is called *worst-case scenario for x*.
- $Z(x)=\max_{y}REGR(x,y)$. Optimal solution is called *worst-case alternative for x*.
- One (might incorrect) assumption is to set top $k$ customers to largest weight, and other customers to smallest weight. And use binary search to find the $k$ makes $\frac{F(s,x)}{F^*(s)}$ largest. ==consider only some of corner points of $S$ ==
	- Given $x$ and integer $0\le k\le n$. Denote the scenario that assigns top $k$ customers to largest weight, and other customers to smallest weight as $s_{x,k}$. $S_x=\{s_{x,k}\}_{k=0,1,\ldots,n}$ . We want to verity that $\max_{s\in S}\frac{F(s,x)}{F^*(s)}=\max_{s\in S_x}\frac{F(s,x)}{F^*(s)}$ for any $x$.
	- Or a simpler version: for any $x,y$, $$\frac{F(w,x)}{F(w,y)}\le \max_{k=1,2,\ldots,n-1}\frac{F(w_k,x)}{F(w_k,y)}$$.
	- #Code 25_both_distance.py
```
Without the plugin it looks like 
<img width="1183" alt="image" src="https://github.com/RyotaUshio/obsidian-math-booster/assets/25512742/57a2b1bd-1552-4f3f-abb0-f03a7ab0d01e">

When enable this plugin, the last equation becomes invisible.
<img width="1202" alt="image" src="https://github.com/RyotaUshio/obsidian-math-booster/assets/25512742/f3a0c68b-98b4-4630-965d-898fd3dd3012">

**Comment by RyotaUshio** [comment]
Hi @Luke2Sky @DDDOH, thanks for reporting.

It was finally fixed in [v1.0.3-beta](https://github.com/RyotaUshio/obsidian-math-booster/releases/tag/1.0.3-beta). 

I knew where the problem came from: a code for [math live preview inside blockquotes](https://ryotaushio.github.io/obsidian-math-booster/math-preview.html#blockquotes).
The challenge was to fix it while keeping this feature alive.

v1.0.3-beta resolves the problem reported in this issue, and I expect the math live preview feature inside blockquote works just like before.
If you find any problems in the new version, please help me by filing a new issue! (You can install this version with BRAT)
